docs/sources/enterprise/query-caching.md

```diff
@@ -27,6 +27,13 @@ You can make a panel retrieve fresh data more frequently by increasing the **Max
 - Reduced API costs.
 - Reduced likelihood that APIs will rate-limit or throttle requests.
 
+## Data sources that work with query caching
+Query caching works for all [Enterprise data sources](https://grafana.com/grafana/plugins/?type=datasource&enterprise=1), and it works for all [built-in data sources]({{< relref "../datasources/_index.md" >}}) except for Elasticsearch, Prometheus, and Loki, which cache queries themselves.
```

Comment:
We probably need to verify this: `it works for all [built-in data sources]({{< relref "../datasources/_index.md" >}}) except for Elasticsearch, Prometheus, and Loki, which cache queries themselves`.

Reply:
Went through and confirmed which built-in data sources display the Cache tab and which ones don't.